contributors/devel/sig-testing/monitoring.md

```diff
@@ -0,0 +1,264 @@
+Test](https://github.com/kubernetes/kubernetes/issues/new?assignees=&labels=kind%2Fflake&template=flaking-test.md)
+
+#### Filling out an issues
+Both test issue templates are reasonably self explanatory so what follows here
+are guidelines and tips on filling out the templates.
+
+When logging a Flaking or Failing test please
+ - use plain text when referring to test names, job names. Inconsistent
+   formatting of names makes it harder to process issues via automation.
+ - keep an eye out for test names that contain markdown parse-able formatting.
+   If you are a test maintainer refrain from including markdown in strings that
+   are used to name your tests, test components etc.
+ 
+#### Fill out the issue for a Flaking Test
+
+1. **Which jobs are flaking**
+
+The job is the tab in TestGrid that you are looking at.
+
+<img src="./testgrid-images/testgrid-jobs.png" height="50%" width="40%">
+
+The above example was taken from the SIG Release dashboard and we can see that
+* `conformance-ga-only` https://testgrid.k8s.io/sig-release-master-blocking#conformance-ga-only,
+* `skew-cluster-latest-kubectl-stable1-gce` https://testgrid.k8s.io/sig-release-master-blocking#skew-cluster-latest-kubectl-stable1-gce,
+* `gci-gce-ingress` https://testgrid.k8s.io/sig-release-master-blocking#gci-gce-ingress,
+* `kind-master-parallel` https://testgrid.k8s.io/sig-release-master-blocking#kind-master-parallel
+
+are flaky 
+
+2. **Which tests are flaking**
+
+Let's grab an example from the SIG release dashboards and look at the
+`node-kubelet-features-master` job in
+https://testgrid.k8s.io/sig-release-master-informing#node-kubelet-features-master.
+
+<img src="./testgrid-images/failed-tests.png" height="70%" width="100%">
+
+Here we see that at 16.07 EDT and 15:07 EDT the job
+```
+[k8s.io] NodeProblemDetector [NodeFeature:NodeProblemDetector] [k8s.io] SystemLogMonitor should generate node condition and events for corresponding errors [ubuntu]
+```
+Failed for Kubernetes commit `9af86e8db` (this value is a row below the time -
+the value above it is the Prow job run ID).
+The corresponding test-infra commit was `11cb57d36` (the value below the commit
+for Kubernetes).
+
+At 15:07 EDT, the job
+```
+[k8s.io] NodeProblemDetector [NodeFeature:NodeProblemDetector] [k8s.io] SystemLogMonitor should generate node condition and events for corresponding errors [cos-stable2]
+```
+
+failed as well.
+
+If one or both of these jobs continue failing, or if they fail frequently
+enough, we should open an issue and investigate.
+
+3. **Since when has it been flaking**
+
+This information you can get from the header of the TestGrid page showing you all the
+tests.
+
+Going from top to bottom, you will see:
+* Time at which the job was started
+* Prow job run ID
+* Kubernetes commit that was tested
+* (Most often) test-infra commit
```

Comment:
```suggestion
* (Optionally) test-infra commit
```

Reply:
Removed "Frequently enough"
Rewrote that list to use @thejoycekung 's red-number-annotated TestGrid diagram (Thanks again JK!)